### DISCUSSION TITLE
分享一个New Bing代理

### DISCUSSION THREAD

**Discussion by socketcat** [PenUniverse/PenMods#179] [奇技淫巧]
签名申请地址：https://proxy.shion.cloudns.org/sydney/create?bundleVersion=1.0
Chathub地址：wss://proxy.shion.cloudns.org/sydney/chathub/

**Comment by tofuture-1** [comment]
想问一下有搭建教程吗？

**Reply by ms-xiao** [reply]
求

**Comment by tofuture-1** [comment]
这个好像挂了

**Reply by nep232** [reply]
没挂

**Reply by tofuture-1** [reply]
OK是我的问题，是还能正常用的

**Reply by JohnXX123456** [reply]
请问，我的显示套接字发生错误，代码：0，是怎么回事
![iwEcAqNqcGcDAQTRDMAF0QmQBrC9xITtLx4dmwXBzAzWWJsAB9IumU7gCAAJomltCgAL0gANs8I jpg_720x720q90](https://github.com/PenUniverse/PenMods-release/assets/160201935/3cc181aa-0abe-4fbe-a42a-03392cd4fc80)

**Reply by tofuture-1** [reply]
你换一个cookie文件试试，这个好像是跟服务器断连了

**Reply by JohnXX123456** [reply]
请问这个代理现在能用吗

**Reply by JohnXX123456** [reply]
我重装penmods之后new bing又用不了了，想请教两个问题:
1.cookies是在cn.bing.com获取还是用魔法在 www.bing.com获取
2.是全部都获取还是只用最后两个
![image](https://github.com/PenUniverse/PenMods-release/assets/160201935/7e8f93c2-ad3e-45cb-bb2f-a98bd5ff44ce)

**Reply by tofuture-1** [reply]
使用电脑的 edge 浏览器，打开如图所示扩展页面（或者通过右边的三个点也可以打开扩展），点击 “打开 Microsoft Edge 加载项”，搜索下载”cookie-Editor“，下载完成后重新打开浏览器，访问 www.bing.com，在该页面再次打开下图所示界面，点击 “Cookie-Editor”，然后点击如图所示按钮，即可将 cookie 复制到剪切板，后在桌面随便新建一个文本文档，将复制内容输入，保存后重命名为 bing-cookies.json 按照词典笔 “关于 cookie” 中的步骤的方法放入词典笔。如果遇到多次发送问题后无反应，请尝试重新导入 cookie 文件，并点击词典笔设置中的 “关于 cookie” 重载文件。
![image](https://github.com/PenUniverse/PenMods-release/assets/80562703/6b2f2c2a-51e2-41c7-8daa-2f02b683e3b2)

![image](https://github.com/PenUniverse/PenMods-release/assets/80562703/ebd78e14-2300-4114-8bf4-83f4e5f24152)

**Reply by tofuture-1** [reply]
最好是用魔法访问www.bing.com，cn的好像有时可以有时不行

**Reply by nep232** [reply]
代理好像炸了

**Reply by JohnXX123456** [reply]
我的现在能用，不过有点慢

**Reply by MGtxzzg** [reply]
为什么我整newbing时按照着教程，打开cn.bing.com复制cookie，再粘贴到文本文档里改名字在复制到笔中再重载cookies，但词典笔一直显示打不开bing-cookies.json文件，当我点一个参考候选（就是自动给的话题），它就显示打不开Json文件，以及Json异常和一大段报告。
图片

![Collage_20240225_073931](https://github.com/PenUniverse/PenMods-release/assets/161115377/fdda08ee-a61b-45ac-bba4-3534d6d70822)

**Reply by tofuture-1** [reply]
你看看名字和路径对没有

**Reply by XiexieLihua** [reply]
如何打开cn.bing.com复制cookie，有教程吗，网上的看不懂？

**Reply by tofuture-1** [reply]
https://www.pbzxkjs.top/?p=248
可以到这里的结尾看

**Reply by XiexieLihua** [reply]
为什么我这里显示的是套接字发生错误，代码：0，一连串好几行

**Reply by tofuture-1** [reply]
用魔法访问www.bing.com获取cookie文件试试

**Reply by XiexieLihua** [reply]
还是不行，变成上面那位的情况了，json异常接一堆乱码

**Reply by tofuture-1** [reply]
登录账号了吗，可以试试用魔法登录之后获取cookie文件

**Reply by nep232** [reply]
我中午的时候也用不了，会报一长串的错，只有早上和晚上可以用

**Reply by XiexieLihua** [reply]
有人可以共享一下可以使用bing chat的bing-cookies.json文件吗？

**Reply by Zhuangmoumou** [reply]
现在这个变成了套接字符串发生错误，代码0了
